autotest.pm

```diff
@@ -231,6 +231,8 @@ sub load_snapshot {
 
 sub _terminate () {
     close $isotovideo;    # uncoverable statement
+    use FindBin '$Bin';
+    chdir $Bin;
```

Comment:
Trying to change to the root source folder just before creating a coverage report from subprocesses does not seem to make a difference. The coverage still reports the same as in before in this PR, with decreased coverage from files that we assume are only significantly covered from subprocesses